org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/JavaElementInfo.java

```diff
@@ -38,4 +38,7 @@ public Object clone() {
 	public IJavaElement[] getChildren() {
 		return JavaElement.NO_ELEMENTS;
 	}
+	public IJavaElement[] getAdvancedChildren() {
```

Comment:
Can you rename this to something more intuitive? Perhaps getExtendedChildren() or even something more specific like getArguments() or getArgumentChildren() if it's only going to be arguments in the list?

Reply:
ok